myflights/apps/core/models.py

```diff
@@ -0,0 +1,128 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django_countries.fields import CountryField
+
+
+class BaseModel(models.Model):
+    """Base abstract model
+    """
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class Airport(BaseModel):
+    """Represents real Airport.
+    """
+
+    name = models.CharField(_('Airport name'), max_length=255, null=True)
+    city_name = models.CharField(_('City name'), max_length=255, null=True, blank=True)
+    country = CountryField(_('Country'))
+    iata = models.CharField(
+        _('3-letter IATA code'), max_length=3, null=True, blank=True
+    )
+    icao = models.CharField(
+        _('4-letter ICAO code'), max_length=4, null=True, blank=True
+    )
+    latitude = models.FloatField(_('Latitude'))
```

Comment:
Why is this needed?

Reply:
Which one, `icao` or `latitude`? Anyways, I've imported most of the columns from archives, complicating model to simply see how to define different fields with different options and mimic more realistic data model (for the sake of training).